src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,54 @@
+package duke;
+
+import java.text.ParseException;
+import java.util.Date;
+import java.text.SimpleDateFormat;
+
+/**
+ * Represents a Deadline object. A <code>Deadline</code> object corresponds to
+ * an agenda to be done by.
+ */
+public class Deadline extends Task {
+    private Date date;
+
+    public Deadline(String description) {
+        super(description);
+        this.type = "D";
```

Comment:
Should be an enum instead of a "magic string"; alternatively, declare a `final static String deadlineType = "D"`.